perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -304,6 +304,17 @@ sub updates_disallowed {
     return $self->next::method(@_);
 }
 
+sub body_disallows_state_change {
+    my $self = shift;
+    my ($problem) = @_;
+    my $c = $self->{c};
+
+    my ($allowed, $body) = $self->per_body_config('update_states_allowed', $problem);
+    $allowed //= 0;
```

Comment:
I assume you want to default to allowed, not disallowed (I think this is what causes the test failure in report_display.t

Reply:
Oops :)